xml/Microsoft.Win32/FileDialog.xml

```diff
@@ -723,7 +723,7 @@
       </Parameters>
       <Docs>
         <param name="e">To be added.</param>
-        <summary>To be added.</summary>
+        <summary>Raises the <see cref="E:System.Windows.FileDialog.FileOk"/> event.</summary>
```

Comment:
```suggestion
        <summary>Raises the <see cref="E:System.Windows.Forms.FileDialog.FileOk"/> event.</summary>
```

Reply:
Maybe this instead?

Reply:
The event you mentioned her belongs to WinForms and not WPF. I have made the changes corresponding to the WPF FileDialog event.